test/libyul/yulOptimizerTests/equalStoreEliminator/mstore_mstore.yul

```diff
@@ -0,0 +1,20 @@
+{
+    let _0 := 0
+    let _32 := 32
+
+    calldatacopy(0, 0, 0x40)
+    let mem32 := mload(_32)
+
+    mstore(_0, mem32)  // Redundant. Overwritten by MSTORE.
```

Comment:
OK, so I guess `mstore8` isn't a candidate for removal, because `mstore` is getting removed here?